pkg/virt-api/webhooks/arm64.go

```diff
@@ -94,23 +129,27 @@ func setDefaultBootloader(vmi *v1.VirtualMachineInstance) {
 	}
 }
 
-// ValidateVirtualMachineInstanceArm64Setting is validation function for validating-webhook
-func ValidateVirtualMachineInstanceArm64Setting(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause {
-	var causes []metav1.StatusCause
-	if cause, ok := verifyInvalidSetting(field, spec); !ok {
-		causes = append(causes, cause)
+// setDefaultDisksBus set default Disks Bus, because sata is not supported by qemu-kvm of Arm64
+func setDefaultDisksBus(vmi *v1.VirtualMachineInstance) {
+	if vmi.Spec.Domain.Devices.Disks != nil {
+		for _, disk := range vmi.Spec.Domain.Devices.Disks {
+			// v1.SetObjectDefaults_VirtualMachineInstance set default bus type as DiskBusSATA, so we reset the bus type to DiskBusSCSI
+			if disk.Disk != nil && (disk.Disk.Bus == "" || disk.Disk.Bus == v1.DiskBusSATA) {
```

Comment:
I think you forgot to remove the SATA override here.

Reply:
Yes, it make sense.